x-pack/plugins/apm/public/components/app/metrics/static_dashboard/index.tsx

```diff
@@ -100,6 +100,7 @@ async function getCreationOptions(
         panels,
         controlGroupInput,
       }),
+      byValueId: 'apmMetricsStaticDashboard',
```

Comment:
This fix requires consumers to provide `byValueId`. The root cause of the problem is that CopyToDashboardModal uses dashboardId to determine if a dashboard is "new" or not. Would it be possible to identify "new" dashboard in such a way that `byValueId` is not needed?

I am concerned that this issue will reamain if consumers do not set 'byValueId'.

Reply:
How about a flag in new dashboard state that signals new dashboards to resolve this issue and anywhere else there is a "new dashboard" check.

We can separate out a solution to the session backups outside of this fix. I don't think they should coupled.

Reply:
That could work. Will close this PR for now then.